### PR TITLE
[UXE-4081] fix: duplicate titles docs

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/build-journey/develop-with-azion/toolsterraform/terraform.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Azion Terraform Provider
+title: Build - Azion Terraform Provider
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/build-journey/develop-with-azion/toolsterraform/terraform.mdx:2]:
```suggestion
title: Build with Azion Terraform Provider
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/build-journey/develop-with-azion/toolsterraform/terraform.mdx:2]: to align with others

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Digital Certificates
+title: Digital Certificates - CHANGE
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:2]:
```suggestion
title: Digital Certificates for Build
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/digital-certificates.mdx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Digital Certificates
+title: Digital Certificates -  change
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/digital-certificates.mdx:2]:
```suggestion
title: Digital Certificates for Secure
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/domains/mtls.mdx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Support for mTLS
+title: Support for mTLS - repeat
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/domains/mtls.mdx:2]:
```suggestion
title: Support for mTLS for Build
```

File: src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/mtls.mdx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Support for mTLS
+title: Support for mTLS - repeat
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/secure/edge-firewall/mtls.mdx:2]:
```suggestion
title: Support for mTLS for Secure
```

File: src/content/docs/en/pages/secure-journey/edit-edge-firewall/edit-main-settings.mdx [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: How to configure main settings
+title: How to configure main settings - check
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/edit-edge-firewall/edit-main-settings.mdx:2]:
```suggestion
title: How to configure Edge Firewall main settings
```

File: src/content/docs/pt-br/homes/cli-home.mdx [hunk 7]
```diff
@@ -12,7 +12,7 @@ product_cards:
   - title: ''
     cards:
       - icon: /assets/docs/images/uploads/icon-overview.svg
-        title: Visão geral
+        title: Azion CLI Visão geral
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/homes/cli-home.mdx:15]:
```suggestion
        title: Visão geral da  Azion CLI
```

File: src/content/docs/pt-br/homes/graphql-home.mdx [hunk 8]
```diff
@@ -10,7 +10,7 @@ product_cards:
   - title: ''
     cards:
       - icon: /assets/docs/images/uploads/icon-overview.svg
-        title: Visão geral
+        title: GraphQL Visão geral
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/homes/graphql-home.mdx:13]:
```suggestion
        title: Visão geral da GraphQL
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-terraform.mdx [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Azion Terraform Provider
+title: Build - Azion Terraform Provider
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/devtools/tools-terraform.mdx:2]:
```suggestion
title: Construa com o Azion Terraform Provider
```

File: src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-overview.mdx [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Visão geral
+title: Frameworks - Visão geral
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/desenvolva-com-azion/guias-frameworks/framework-overview.mdx:2]:
```suggestion
title: Visão geral dos Frameworks
```

File: src/content/docs/pt-br/pages/build-jornada/visao-geral/visao-geral.mdx [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Visão geral
+title: Jornada de Build - Visão geral
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/build-jornada/visao-geral/visao-geral.mdx:2]:
```suggestion
title: Visão geral de Build 
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Digital Certificates
+title: Digital Certificates - CHANGE
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:2]:
```suggestion
title: Digital Certificates para Build
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/domains/mtls.mdx [hunk 13]
```diff
@@ -1,6 +1,5 @@
 ---
-
-title: Suporte para mTLS
+title: Suporte para mTLS - change?
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/domains/mtls.mdx:2]:
```suggestion
title: Suporte para mTLS para Build
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/digital-certificates.mdx [hunk 14]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Digital Certificates
+title: Digital Certificates - change?
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/digital-certificates.mdx:2]:
```suggestion
title: Digital Certificates para Secure
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/mtls.mdx [hunk 15]
```diff
@@ -1,6 +1,6 @@
 ---
 
-title: Suporte para mTLS
+title: Suporte para mTLS - change?
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/referencia/secure/edge-firewall/mtls.mdx:3]:
```suggestion
title: Suporte para mTLS para Secure
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**